### PR TITLE
GODRIVER-2435 Add comment option tests for distinct helper.

### PR DESCRIPTION
**Pull request by qingyang-hu**
[GODRIVER-2435](https://jira.mongodb.org/browse/GODRIVER-2435)
Add comment option tests for distinct helper.

### CODE REVIEW

File: mongo/options/distinctoptions.go [hunk 1]
```diff
@@ -15,6 +15,10 @@ type DistinctOptions struct {
 	// default value is nil, which means the default collation of the collection will be used.
 	Collation *Collation
 
+	// A string or document that will be included in server logs, profiling logs, and currentOp queries to help trace
+	// the operation.  The default value is nil, which means that no comment will be included in the logs.
```
Comment (prestonvasquez) [inline mongo/options/distinctoptions.go:19]:
```suggestion
	// the operation. The default value is nil, which means that no comment will be included in the logs.
```
Based on past reviews and the code, it seems we prefer single spaces after punctuation.

Comment (prestonvasquez) [inline mongo/options/distinctoptions.go:19]:
```suggestion
	// the operation. The default value is nil, which means that no comment will be included in the logs.
```
Based on past reviews and from what I've seen in the code, it seems we prefer single spaces after punctuation.

**Review by prestonvasquez — APPROVED** [review]
LGTM!  Just one small observation.

File: mongo/options/distinctoptions.go (hunk 1 above)
Comment (benjirewis) [reply mongo/options/distinctoptions.go:19]: Oh do some Go libraries use double spaces? Is there a reason to not use single spaces?

**Review by benjirewis — APPROVED** [review]
Looks good thanks!

File: mongo/options/distinctoptions.go (hunk 1 above)
Comment (qingyang-hu) [reply mongo/options/distinctoptions.go:19]: Good catch! It's my typo. Our codebase usually follows the single-space convention, although I did see some counter examples. I also read a golang style guide (maybe from uber?) just a few weeks ago, which required single-space as I can remember, but I failed to retrieve the guide online.

**Pull request merged**